TEMPLATE.md

```diff
@@ -0,0 +1,81 @@
+---
+id:
+title:
+status: draft
+created:
+contributors:
+    -
+---
+
+## Preamble
+
+    Idea: <to be assigned>
+    Title: <title>
+    Status: <Draft | In Progress | Closed >
+    Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+    Requires (*optional): <Idea number(s)>
+    Replaces (*optional): <Idea number(s)>
+
+
+## Summary
+<!-- "If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the Idea. -->
+
+## Swarm Participants
+<!-- Each contributor pledges to the idea with their FOCUS value. (hours per week) -->
+<!-- Here all roles in swarm are defined and filled, one of the contributors should responsibility of the Idea as Lead. -->
+
+<!-- Testing/Evaluation support role is also mandatory to check in on specified Goal dates or earlier. -->
+
+<!-- Lead Contributor is the Owner of the Idea. If required, they can get support from a PM, but should be responsible for end to end execution of the Idea. This includes ensuring appropriate resources are allocated, setting realistic timelines and milestones, and any post-launch metrics or bug fixes that are attributed to the Idea -->
+<!-- A swarm requires at minimum 3 contributors and 1 evaluator/tester -->
+<!-- 'Contributor' should be replaced with a descriptive role type. -->
+- Lead Contributor: <!-- @username -->
+- Testing & Evaluation: <!-- @username -->
+- Contributor: <!-- @username -->
+- Contributor: <!-- @username -->
+- PM: <!--- @username -->
+- UX (if relevant): <!-- @username -->
+<!-- - Contributor: @username -->
+
+## Product Overview
+<!-- A short (~200 word) description and motivation of the Idea. Without clear explanation the Idea should not proceed. Can include User Stories -->
+<!-- Testing/Evaluation role accepts responsbility to checkin at Goal dates, -->
+<!-- forces discussion to continue implementation or recommend disband and post-mortem. -->
+
+### Product Description
+<!-- What functionality are you adding? What will this look like from a user perspective? Why is this important? -->
+
+### Requirements & Dependencies
+<!-- Are there bugs or feature requests in other repositories that are part of this Idea? -->
+<!-- There is no approval unless the idea requires to be reviewed by supporting organelles (Financial, Hiring, or Design). -->
+<!-- The Swarm must develop a fully fleshed out Requirements document for the idea to proceed, to the satisfaction of participants. -->
```

Comment:
The need for "requirement document" is confusing to me. Is this not part of the idea and the headings outlined here to complete? Why the need for a different document?

Reply:
we aren't using this now so removing